cmd/setup.go

```diff
@@ -59,6 +76,27 @@ func runSetupError(cmd *cobra.Command, args []string) error {
 		return microerror.Mask(err)
 	}
 
+	k8sApiUrl := os.Getenv(EnvVarK8sApiUrl)
```

Comment:
I don't think it's a good idea to use this env vars here, we are passing all the options as flags.